napari/_qt/qt_main_window.py

```diff
@@ -74,8 +83,14 @@ def __init__(self, viewer: 'Viewer', parent=None) -> None:
         super().__init__(parent)
         self._ev = None
         self.qt_viewer = QtViewer(viewer, show_welcome_screen=True)
-
         self._quit_app = False
+        self._initial_theme = get_system_theme()
```

Comment:
Had to use this logic cause I could not get a Qt event to work in detecting it 🤔

Also I moved this here, so that it could work when running napari from ipython et al.

Reply:
I was wondering when I watched that video how you did that!  😂   I think a 1-second poll for something that nearly never happens feels a bit too much.  lacking a system event, I'm more than happy with this happening once on startup

Reply:
Yeah I felt the same way, but wanted to try it out :)